### DISCUSSION TITLE
Uninstallation?

### DISCUSSION THREAD

**Discussion by scottonanski** [lllyasviel/Fooocus#1362] [General]
I'm curious if there's a specific way to uninstall this? I plan on reinstalling it, but I don't want a bunch of stuff littering my file system until I do.

Do I have to go through the requirements_version.txt file and uninstall each package, and then just delete the direction where I cloned the repo?

```
pip3 uninstall -y torchsde
pip3 uninstall -y einops
pip3 uninstall -y transformers
pip3 uninstall -y safetensors
pip3 uninstall -y accelerate
pip3 uninstall -y pyyaml
pip3 uninstall -y Pillow
pip3 uninstall -y scipy
pip3 uninstall -y tqdm
pip3 uninstall -y psutil
pip3 uninstall -y numpy
pip3 uninstall -y pytorch_lightning
pip3 uninstall -y omegaconf
pip3 uninstall -y gradio
pip3 uninstall -y pygit2
pip3 uninstall -y opencv-contrib-python
pip3 uninstall -y httpx

```

Because `pip3 uninstall Fooocus` doesn't do anything.

Please advise.

Thanks.

ps - I'm running Linux.

**Comment by lllyasviel** [comment]
hi,

you can delete git cloned folder and remove the conda env or python venv

**Reply by scottonanski** [reply]
Thanks.

(Edit)

**For anyone else coming across this:**

I installed by way of native system Python;

```
git clone https://github.com/lllyasviel/Fooocus.git
cd Fooocus
pip3 install -r requirements_versions.txt
```

So the conda environment _(conda env)_ or the python virtual environement _(python venv)_ wouldn't apply here.

**Reply by lllyasviel** [reply]
Removing a Conda environment or a Python virtual environment (venv) on a Linux system is a straightforward process. Here's how you can do it for each type of environment:

### For Conda Environment

1. **List Existing Environments**: First, it's helpful to know the name of the environment you want to remove. List all Conda environments with:
   ```bash
   conda env list
   ```
   or 
   ```bash
   conda info --envs
   ```

2. **Remove Environment**: Once you know the name of the environment, you can remove it using the following command:
   ```bash
   conda env remove --name your_env_name
   ```
   Replace `your_env_name` with the name of the Conda environment you want to delete.

### For Python Virtual Environment (venv)

1. **Deactivate Environment**: If you're currently in the virtual environment, you need to deactivate it first. Run:
   ```bash
   deactivate
   ```

2. **Remove Environment Directory**: The virtual environment is just a directory containing all the necessary files. You can delete this directory with the `rm` command. For example, if your virtual environment is named `venv`, you would use:
   ```bash
   rm -rf venv/
   ```
   Replace `venv/` with the name of your virtual environment directory.

### Additional Notes

- Be careful with the `rm -rf` command in Linux, as it will permanently delete files without any confirmation.
- It's important to deactivate the virtual environment before trying to remove it to avoid any potential conflicts.
- After removing a Conda environment, you might want to run `conda clean --all` to clean up any unused packages and caches.

**Reply by scottonanski** [reply]
Awesome!

I installed using native system Python, so the removal is straightforward at this point; but thanks for taking the time to inform me of these other options in the future!
